### PR TITLE
[Music] Levelbuilder quick wins

### PR DESCRIPTION
**Pull request by mikeharv**
This adds the following changes/improvements to the level edit page for Music Lab levels:

This lower warning has been removed as it is no longer needed or accurate:
**Before:**
image
**After:**
image

A warning has been added to help clarify limitations around loading libraries with Lab2. Additionally, the dropdown menu will now select the recommended 'launch2024' by default, and populate its allowed sounds list, unless the level has already been configured with a library.

**Before:**
image
**After:**
image

## PR Checklist:

- [ ] Tests provide adequate coverage
- [ ] Privacy and Security impacts have been assessed
- [ ] Code is well-commented
- [ ] New features are translatable or updates will not break translations
- [ ] Relevant documentation has been added or updated
- [ ] User impact is well-understood and desirable
- [ ] Pull Request is labeled appropriately
- [ ] Follow-up work items (including potential tech debt) are tracked and linked

### CODE REVIEW

File: apps/src/lab2/levelEditors/levelData/EditMusicLevelData.tsx [hunk 1]
```diff
@@ -39,6 +40,10 @@ const EditMusicLevelData: React.FunctionComponent<EditMusicLevelDataProps> = ({
   }, []);
 
   const [levelData, setLevelData] = useState(initialLevelData);
+  // Immediately set a level, if needed, so we can populate its allowed sounds.
+  if (!levelData.library) {
+    levelData.library = RECOMMENDED_LIBRARY;
```
Comment (sanchitmalhotra126) [inline apps/src/lab2/levelEditors/levelData/EditMusicLevelData.tsx:45]: Just confirming that this triggers the `useEffect` to load the library? I would think that we need to use a state update like `setLevelData({...levelData, library: RECOMMENDED_LIBRARY})` but maybe not?

**Review by sanchitmalhotra126 — APPROVED** [review]
Nice improvements!

File: apps/src/lab2/levelEditors/levelData/EditMusicLevelData.tsx (hunk 1 above)
Comment (mikeharv) [reply apps/src/lab2/levelEditors/levelData/EditMusicLevelData.tsx:45]: It does!

File: apps/src/lab2/levelEditors/levelData/EditMusicLevelData.tsx [hunk 2]
```diff
@@ -39,6 +40,10 @@ const EditMusicLevelData: React.FunctionComponent<EditMusicLevelDataProps> = ({
   }, []);
 
   const [levelData, setLevelData] = useState(initialLevelData);
+  // Immediately set a level, if needed, so we can populate its allowed sounds.
```
Comment (breville) [inline apps/src/lab2/levelEditors/levelData/EditMusicLevelData.tsx:43]: nit: "set a library"?

Comment (mikeharv) [reply apps/src/lab2/levelEditors/levelData/EditMusicLevelData.tsx:43]: Ha, yep. Will fix next week.

**Pull request merged**